e2e/cypress/tests/integration/enterprise/channel/channel_groups_spec.ts

```diff
@@ -31,7 +31,7 @@ describe('channel groups', () => {
         cy.apiUpdateConfig({LdapSettings: {Enable: true}, ServiceSettings: {EnableTutorial: false}});
 
         // # Create a new team and associate one group to the team
-        cy.apiCreateTeam('team', 'Team').then(({team}) => {
+        cy.apiCreateTeam('team', 'Team').then((team) => {
```

Comment:
This test failed. Please update to latest, then this can be reverted.

Reply:
Done! TY 👍